modules/storages/app/components/storages/admin/side_panel/validation_result_component.rb

```diff
@@ -31,29 +31,42 @@
 module Storages
   module Admin
     module SidePanel
-      class ValidationResultComponent < ApplicationComponent # rubocop:disable OpenProject/AddPreviewForViewComponent
+      class ValidationResultComponent < ApplicationComponent
         include OpPrimer::ComponentHelpers
         include OpTurbo::Streamable
 
-        def initialize(result:)
-          super(result)
+        def initialize(storage:, result:)
+          super(storage)
           @result = result
         end
 
         private
 
-        def status_indicator
-          case @result.type
-          when :healthy
-            { scheme: :success, label: I18n.t("storages.health.label_healthy") }
-          when :warning
-            { scheme: :attention, label: I18n.t("storages.health.label_warning") }
-          when :error
-            { scheme: :danger, label: I18n.t("storages.health.label_error") }
+        def summary_header
+          tally = @result.tally
+          failure = tally[:failure] || 0
+          warning = tally[:warning] || 0
+
+          if failure > 0
+            { icon: :alert, icon_color: :danger, text: I18n.t("storages.health.checks.failures", count: failure) }
+          elsif warning > 0
+            { icon: :alert, icon_color: :attention, text: I18n.t("storages.health.checks.warnings", count: warning) }
           else
-            raise ArgumentError, "Unknown validation result type for status indicator: #{@result.type}"
+            { icon: :"check-circle", icon_color: :success, text: I18n.t("storages.health.checks.failures", count: 0) }
           end
         end
+
+        def summary_description
+          text = if @result.healthy?
+                   I18n.t("storages.health.summary.success")
+                 elsif @result.unhealthy?
+                   I18n.t("storages.health.summary.failure")
+                 else
+                   I18n.t("storages.health.summary.warning")
+                 end
+
+          "#{text} #{I18n.t('storages.health.checked', datetime: helpers.format_time(@result.latest_timestamp))}"
```

Comment:
:red_circle: Same as for the other translation composed this way: Languages have different needs on ordering their sentences. Additionally I'd rename the input `text` to help translators understand the context better:

```suggestion
          I18n.t("storages.health.checked", health_check_text:, datetime: helpers.format_time(@result.latest_timestamp))
```

Reply:
In this case, I think it is fine to leave as is. This "label" will go at the top of the page listing the last status and when it was run. ("Success. Last checked on: 1920-02-30 25:61)

It isn't as much of a sentence than 2 values together.

Reply:
True. It does even translate to two sentences. In that case it's fine, though maybe arguable whether the component should then compose the two things together in a string, rather than the erb rendering it like that.

But yeah, that comment is clearly not a :red_circle: anymore :)

Reply:
I agree that it might be weird to have those concatenated. But they ARE two different locales, just put into the same <p>.